cms/djangoapps/contentstore/management/commands/tests/test_import.py

```diff
@@ -44,38 +39,23 @@ def setUp(self):
         self.content_dir = path(tempfile.mkdtemp())
         self.addCleanup(shutil.rmtree, self.content_dir)
 
-        # Create good course xml
-        self.good_dir = self.create_course_xml(self.content_dir, self.BASE_COURSE_KEY)
+        self.base_course_key = self.store.make_course_key(u'edX', u'test_import_course', u'2013_Spring')
+        self.truncated_key = self.store.make_course_key(u'edX', u'test_import', u'2014_Spring')
 
-        # Create run changed course xml
-        self.dupe_dir = self.create_course_xml(self.content_dir, self.DIFF_KEY)
+        # Create good course xml
+        self.good_dir = self.create_course_xml(self.content_dir, self.base_course_key)
 
         # Create course XML where TRUNCATED_COURSE.org == BASE_COURSE_ID.org
         # and BASE_COURSE_ID.startswith(TRUNCATED_COURSE.course)
-        self.course_dir = self.create_course_xml(self.content_dir, self.TRUNCATED_KEY)
+        self.course_dir = self.create_course_xml(self.content_dir, self.truncated_key)
 
     def test_forum_seed(self):
         """
         Tests that forum roles were created with import.
         """
-        self.assertFalse(are_permissions_roles_seeded(self.BASE_COURSE_KEY))
-        call_command('import', self.content_dir, self.good_dir)
-        self.assertTrue(are_permissions_roles_seeded(self.BASE_COURSE_KEY))
-
-    def test_duplicate_with_url(self):
```

Comment:
This test is bogus with split, correct? It is possible now to create 2 different courses that differ only by run.